### PR TITLE
Account management improvements

### PR DESCRIPTION
**Pull request by AlokSwaminathan**

### CODE REVIEW

**Review by BrilliantDeviation7 — APPROVED** [review]
I've fixed any issues I found; rest looks good to me.

**Comment by manningwu07** [comment]
Code logic looks fine and styling stuff can be tuned later; can you make sure to fix/patch all deploy errors from this PR? I can then approve merging afterwards.

**Review by manningwu07 — DISMISSED** [review]
Code logic looks fine and styling stuff can be tuned later

can you make sure to fix/patch all deploy errors from this PR? I can then approve merging afterwards.

File: src/lib/auth.ts [hunk 1]
```diff
@@ -137,26 +144,6 @@ export const useAuthHandlers = () => {
     }
   };
 
-  const signInWithGoogle = async () => {
```
Comment (BrilliantDeviation7) [inline src/lib/auth.ts:140]: Bring back `signInWithGoogle` here and in `src\app\login\page.tsx`. Removing this breaks the login. We can clean it up later, but for now let's just keep this here.

File: src/lib/auth.ts [hunk 2]
```diff
@@ -172,11 +159,31 @@ export const useAuthHandlers = () => {
...
+
+  const undoEmailChange = async (code: string) => {
+    await applyActionCode(auth, code);
+  };
+
   return {
     signUpWithEmail,
     signInWithEmail,
     signUpWithGoogle,
-    signInWithGoogle,
```
Comment (BrilliantDeviation7) [inline src/lib/auth.ts:179]: Keep this (see first comment).

**Review by BrilliantDeviation7 — CHANGES_REQUESTED** [review]

**Review by BrilliantDeviation7 — APPROVED** [review]

**Pull request merged**